### ISSUE TITLE
Don't see coupled congestion in allowed_congestion_control

### ISSUE THREAD

**Issue by vvinothkumar** [multipath-tcp/mptcp#62]
Hello

I compiled and installed 0.89.2 on x86. In my config file, I have coupled enabled by default. But after booting, I did not find the coupled. Only change I made on mptcp kernel config, I disabled USB 3.0. 

$ uname -a
Linux nuc 3.14.19 #2 SMP Wed Nov 12 16:06:00 IST 2014 x86_64 x86_64 x86_64 GNU/Linux

$ sysctl -a | grep congestion
net.ipv4.tcp_allowed_congestion_control = cubic reno
net.ipv4.tcp_available_congestion_control = cubic reno lia olia wvegas
net.ipv4.tcp_congestion_control = cubic 

$ cat config-3.14.19 | grep TCP 
CONFIG_INET_TCP_DIAG=m
CONFIG_TCP_CONG_ADVANCED=y
CONFIG_TCP_CONG_BIC=m
CONFIG_TCP_CONG_CUBIC=y
CONFIG_TCP_CONG_WESTWOOD=m
CONFIG_TCP_CONG_HTCP=m
CONFIG_TCP_CONG_HSTCP=m
CONFIG_TCP_CONG_HYBLA=m
CONFIG_TCP_CONG_VEGAS=m
CONFIG_TCP_CONG_SCALABLE=m
CONFIG_TCP_CONG_LP=m
CONFIG_TCP_CONG_VENO=m
CONFIG_TCP_CONG_YEAH=m
CONFIG_TCP_CONG_ILLINOIS=m
CONFIG_TCP_CONG_COUPLED=y
CONFIG_TCP_CONG_OLIA=y
CONFIG_TCP_CONG_WVEGAS=y
CONFIG_DEFAULT_TCP_CONG="cubic"
CONFIG_TCP_MD5SIG=y
CONFIG_MPTCP=y
CONFIG_MPTCP_PM_ADVANCED=y
CONFIG_MPTCP_FULLMESH=y
CONFIG_MPTCP_NDIFFPORTS=y
CONFIG_MPTCP_BINDER=m
CONFIG_DEFAULT_MPTCP_PM="fullmesh"
CONFIG_MPTCP_SCHED_ADVANCED=y
CONFIG_MPTCP_ROUNDROBIN=m
CONFIG_DEFAULT_MPTCP_SCHED="default"
CONFIG_NETFILTER_XT_TARGET_TCPMSS=m
CONFIG_NETFILTER_XT_TARGET_TCPOPTSTRIP=m
CONFIG_NETFILTER_XT_MATCH_TCPMSS=m
CONFIG_IP_VS_PROTO_TCP=y
CONFIG_RDS_TCP=m
CONFIG_NET_TCPPROBE=m
CONFIG_ISCSI_TCP=m
CONFIG_ATM_TCP=m

**Comment by gdetal** [comment]
Hello,

coupled has been recently renamed to lia.

**Comment by vvinothkumar** [comment]
Please tell me the reason behind why the default congestion control is changed from coupled(lia) to cubic?

**Comment by gdetal** [comment]
Ok, it seems that the Kconfig was not updated and thus still tries to set "coupled" as default cc while it should be "lia".

However, I don't know why it has put cubic as the default cc but you can update your .config by replacing CONFIG_DEFAULT_TCP_CONG="lia"

**Comment by vvinothkumar** [comment]
Okay. Thanks :-) And what is the best congestion control for a system having 8 BDP?

**Comment by gdetal** [comment]
What do you mean by 8 BDP ? 8 subflows following the same path ?

**Comment by vvinothkumar** [comment]
Yes, 8 PPP connections.

**Comment by cpaasch** [comment]
Fixed by:

commit 05426c5fdfd3f47797273fe6ddca67dda7eaeb3e
Author: Gregory Detal gregory.detal@uclouvain.be
Date:   Tue Dec 9 17:23:00 2014 +0100

```
mptcp: fix lia Kconfig

Change all references to coupled to lia.

Fixes: 28b17c26770b (mptcp: Change name of Coupled to Lia)
Signed-off-by: Gregory Detal <gregory.detal@uclouvain.be>
Signed-off-by: Christoph Paasch <christoph.paasch@gmail.com>
```
